src/shared-components/Navigation/TopNavigation/TransactionTypeNavigation.tsx

```diff
@@ -0,0 +1,40 @@
+import { Badge, Stack, Typography } from '@mui/material';
+import FilterListIcon from '@mui/icons-material/FilterList';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import { theme } from 'src/styles/theme';
+import { colors } from 'src/styles/colors';
+
+interface TransactionTypeNavigationProps {
+  month: string;
```

Comment:
This should be 'interval' instead of 'month' since we want to show what selected interval is.

Reply:
Fixed